### PR TITLE
Sorted imports in hanging grid style

### LINKED ISSUE #12929: Sort imports
Sort imports in the package uniformly (see possible config in #12928) and introduce tests to check the files: `isort -rc -c -df` in CI.

### PR DESCRIPTION
**Pull request by jirikuncar**
cherry-picked from #12970 
closes #12929
closes #12935

### CODE REVIEW

File: sympy/physics/units/definitions.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
-from sympy import pi, Rational, sqrt
+from sympy import Rational, pi, sqrt
 from sympy.physics.units import Quantity
-from sympy.physics.units.dimensions import length, mass, force, energy, power, pressure, frequency, time, velocity, \
-    impedance, voltage, conductance, capacitance, inductance, charge, magnetic_density, magnetic_flux, current, action, \
-    amount_of_substance, luminous_intensity, temperature, acceleration
-from sympy.physics.units.prefixes import kilo, milli, micro, nano, pico, deci, centi
+from sympy.physics.units.dimensions import (
```
Comment (asmeurer) [inline sympy/physics/units/definitions.py]: Why do you always have the ( on an empty line? Why not 

```py
ffrom sympy.physics.units.dimensions import (acceleration, action,
     amount_of_substance, capacitance, charge, conductance, current, energy,
     force, frequency, impedance, inductance, length, luminous_intensity,
     magnetic_density, magnetic_flux, mass, power, pressure, temperature,
     time, velocity, voltage)
```

Comment (jirikuncar) [inline sympy/physics/units/definitions.py]: You can choose between different line output modes in https://github.com/timothycrosley/isort#multi-line-output-modes

I guess you want something between 0 and 4 like:

```python
from foo.bar.baz import (abc,
    def, ghi, klm)
```
but then the next line should start with opening bracket.

Other alternative is "hanging indent", but somebody told me that the open bracket style is used more widely in SymPy.

```python
from third_party import \
    lib1, lib2, lib3, \
    lib4, lib5, lib6
```

**Comment by jirikuncar** [comment]
Travis fail on single build seems unrelated. Can anyone restart the job?

```
sympy/combinatorics/tests/test_homomorphisms.py[1] 
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated
```

**Comment by smichr** [comment]
> closes #12929
> closes #12935

Do you intend to close the entire issues? As I understand it, this PR is just addressing one module. Is that your intent?

**Comment by jirikuncar** [comment]
@smichr yes, we can close them. I don't see much interest for global sort.

**Comment by smichr** [comment]
I can see the value of it. It's the sort of thing you do once and then it's done. There is added value for anyone editing the code. As I recall, however, implementing globally raised circular imports. But this or need not wait on that issue.

**Pull request merged**